monailabel/utils/others/generic.py

```diff
@@ -216,3 +219,72 @@ def is_openslide_supported(name):
     if ext and ext in supported_ext:
         return True
     return False
+
+
+def get_bundle(app_dir, conf):
```

Comment:
Please change to use the MONAI API instead:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/bundle/scripts.py#L362

Thanks.